hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/RolloutListGrid.java

```diff
@@ -198,6 +220,10 @@ private void handleEvent(final RolloutChangeEvent rolloutChangeEvent) {
                 .setValue(new RolloutRendererData(rollout.getName(), rollout.getStatus().toString()));
     }
 
+    private static boolean rolloutIsDeleted(final Rollout rollout) {
+        return rollout == null;
```

Comment:
The rollout can not be `null`. The `findRolloutWithDetailedStatus` throws a EntityNotFoundException if the rollout is null. I think you have to check the `delete` flag, or?